src/components/login/login.test.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { Login, mapStateToProps } from '.';
+
+const state = {
+  login: { message: '', status: 200, token: 'mckndjvjdsfv' },
+  apiCallsInProgress: 0,
+};
+
+const renderComponent = (args) => {
+  const defaultProps = {
+    loginAction: jest.fn(() => {}),
+    message: '',
+    loading: 0,
+    history: {},
+  };
+  const props = { ...defaultProps, ...args };
+  return shallow(<Login {...props} />);
```

Comment:
How are we testing interactions with child components in these test cases?

Reply:
I included the integration tests with mount.